backend/src/test/java/de/neuefische/paulkreft/backend/game/service/GameServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package de.neuefische.paulkreft.backend.game.service;
+
+import de.neuefische.paulkreft.backend.game.model.Game;
+import de.neuefische.paulkreft.backend.game.model.GameCreate;
+import de.neuefische.paulkreft.backend.game.repository.GameRepo;
+import de.neuefische.paulkreft.backend.services.IdService;
+import de.neuefische.paulkreft.backend.services.TimeService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+class GameServiceTest {
+
+    private GameRepo gameRepo;
+    private GameService gameService;
+    private IdService idService;
+    private TimeService timeService;
+
+    @BeforeEach
+    void setUp() {
+        gameRepo = mock(GameRepo.class);
```

Comment:
Du könntest diese Zuweisung schon in der Klassen Deklaration machen. Also: private final GameRepo gameRepo = mock(GameRepo.class);

Reply:
done 👍